mm2src/mm2_main/src/lp_wallet.rs

```diff
@@ -413,40 +413,45 @@ pub struct GetMnemonicResponse {
     pub mnemonic: MnemonicForRpc,
 }
 
-#[derive(Debug, Display, Serialize, SerializeErrorType)]
+#[derive(Debug, Display, Serialize, SerializeErrorType, EnumFromStringify)]
 #[serde(tag = "error_type", content = "error_data")]
-pub enum GetMnemonicError {
+pub enum WalletsStorageRpcError {
     #[display(fmt = "Invalid request error: {}", _0)]
```

Comment:
Won't this API error renaming be a breaking change for the GUI?

Reply:
I doubt since error_type and error_data extracts the type and data from the error.. cc @CharlVS

Reply:
Yeah, looks like error enum variant names are important as GUI relies on them